Some updates to v3 (#69)
ankitects/anki-manual: src/studying.md

subdecks it contains.

On the decks screen, your decks will be displayed in a list. There are
three columns. 'Due' is the count of waiting
reviews. 'New' is the number of new
cards that are ready to be learnt that day. The third column shows

> the number of cards currently in learning.

Reviewer: Is  this order correct?

Author: Corrected. The second column is unlabelled, perhaps label it as "Learn" could be helpful, specially for new users.

You can click the gears button to the right of a deck to rename or
delete a deck, change its options, or [export](exporting.md) it.

With the v1 scheduler, when a deck has subdecks, the cards will appear from [each deck in
turn](studying.md#display-order).

From the [v2 scheduler](https://faqs.ankiweb.net/the-anki-2.1-scheduler.html),

> when a deck has subdecks, reviews are taken from all children decks

Reviewer: In v3, doesn't this depend on the configured options?

Author: Better this way?

```
- With the [v2 scheduler](https://faqs.ankiweb.net/the-anki-2.1-scheduler.html),
when a deck has subdecks, reviews are taken from all children decks
at once. The review limit of the child decks is ignored - only the limit of the
deck you clicked on applies.

- With the [v3 scheduler](https://faqs.ankiweb.net/the-2021-scheduler.html)
each child deck's limit is also enforced, and you do not need to see the cards 
in deck order either. See the [deck options](deck-options.md#review-sort-order) section of the manual. 
```

You can click the gears button to the right of a deck to rename or
delete a deck, change its options, or [export](exporting.md) it.

- With the v1 scheduler, when a deck has subdecks, the cards will appear from [each deck in
turn](studying.md#display-order).

- With the [v2 scheduler](https://faqs.ankiweb.net/the-anki-2.1-scheduler.html),
when a deck has subdecks, reviews are taken from all children decks
at once. The review limit of the child decks is ignored - only the limit of the
deck you clicked on applies.

- With the [v3 scheduler](https://faqs.ankiweb.net/the-2021-scheduler.html)
each child deck's limit is also enforced, and you do not need to see the cards 

> in deck order either. See the [deck options](deck-options.md#review-sort-order) section of the manual. 

Reviewer: ```suggestion
in deck order either. See the [deck options](deck-options.md#review-sort-order) section of the manual for more information.
```

**Easy** immediately converts the card into a review card, even if there
were steps remaining. By default, the card will be shown again 4 days
later, and then at increasingly long delays. In the v1 scheduler, the "Easy" button will not be
shown if you are in relearning mode and it would give the same interval

> as “Good.” With the [v2 scheduler](https://faqs.ankiweb.net/the-anki-2.1-scheduler.html),

Reviewer: v2+?

Author: Corrected

**Easy** immediately converts the card into a review card, even if there
were steps remaining. By default, the card will be shown again 4 days
later, and then at increasingly long delays. In the v1 scheduler, the "Easy" button will not be

> shown if you are in relearning mode and it would give the same interval

Reviewer: ```suggestion
shown if you are in relearning mode as it would give the same interval
```

As with learning cards, you can use 1-4 on the keyboard to select an
answer. Pressing the spacebar will select 'Good'.

See [Deck Options](deck-options.md) and the [FAQ](https://faqs.ankiweb.net/what-spaced-repetition-algorithm.html)
to learn more about how the algorithm works. 

## Due Counts and Fuzz Factor

When only the question is shown, Anki shows three numbers like 12 + 34 +
56 at the bottom of the screen. These represent the new cards, cards in
learning, and cards to review. If you’d prefer not to see the numbers,
you can turn them off in Anki’s preferences.

In the v1 scheduler, the numbers count _reviews_ needed to finish all the
cards in that queue, not the number of _cards_. If you have multiple
steps configured for lapsed cards, the number will increase by more than
one when you fail a card, since that card needs to be shown several times.

> From the v2scheduler, the numbers count _cards_, so the number will always

Reviewer: ```suggestion
From the v2 scheduler, the numbers count _cards_, so the number will always
```

increase by one regardless of the steps remaining.

When the answer is shown, Anki shows an estimate of the next time a card
will be shown above each button. If you’d prefer not to see the
estimates, you can disable them in Anki’s [preferences](preferences.md).

Anki additionally adds a small amount of random variation (Fuzz Factor) to the next
due times, in order to prevent cards that were introduced together and
always rated the same from always staying next to each other. From 2.1.45+, 

> this variation is reflected on the answer buttons, instead of only being 

Reviewer: ```suggestion
this variation is reflected on the answer buttons.
```

answer. Pressing the spacebar will select 'Good'.

See [Deck Options](deck-options.md) and the [FAQ](https://faqs.ankiweb.net/what-spaced-repetition-algorithm.html)
to learn more about how the algorithm works. 

## Due Counts and Fuzz Factor

When only the question is shown, Anki shows three numbers like 12 + 34 +
56 at the bottom of the screen. These represent the new cards, cards in
learning, and cards to review. If you’d prefer not to see the numbers,
you can turn them off in Anki’s preferences.

In the v1 scheduler, the numbers count _reviews_ needed to finish all the
cards in that queue, not the number of _cards_. If you have multiple
steps configured for lapsed cards, the number will increase by more than
one when you fail a card, since that card needs to be shown several times.

From the v2 scheduler, the numbers count _cards_, so the number will always
increase by one regardless of the steps remaining.

When the answer is shown, Anki shows an estimate of the next time a card
will be shown above each button. If you’d prefer not to see the
estimates, you can disable them in Anki’s [preferences](preferences.md).

> Anki additionally adds a small amount of random variation (Fuzz Factor) to the next

Reviewer: ```suggestion
Anki additionally adds a small amount of random variation ("fuzz") to the next
```

due times, in order to prevent cards that were introduced together and

> always rated the same from always staying next to each other. From 2.1.45+, 

Reviewer: ```suggestion
always rated the same from always staying next to each other. In the v3 scheduler, 
```

this variation is reflected on the answer buttons.

> applied when answering, as in previous versions. 

Reviewer: Delete this line?

Author: Deleted, sorry!
